oidc-controller/api/templates/verified_credentials.html

```diff
@@ -333,193 +274,261 @@ <h1 class="mobile-device my-1">Continue with:</h1>
           <a
             title="Download BC Wallet"
             href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet"
-            >Download the BC Wallet app
+          >Download the BC Wallet app
           </a>
         </p>
       </div>
     </div>
-  </body>
-  <script>
-    /**
-     * Function to detect the user's browser
-     */
-    function getBrowser() {
-      var userAgent = navigator.userAgent || navigator.vendor;
-
-      if (/android/i.test(userAgent)) {
-        return "Android";
-      }
-
-      if (
-        /iPad|iPhone|iPod/.test(userAgent) ||
-        (/Macintosh/.test(userAgent) && "ontouchend" in document) ||
-        (navigator.platform === "MacIntel" && navigator.maxTouchPoints > 1) ||
-        (navigator.vendor && navigator.vendor.indexOf("Apple") > -1)
-      ) {
-        return "iOS";
-      }
-
-      return "unknown";
-    }
-    function showHideElementForBrowser(elementId, show) {
-      if (getBrowser() === "Android" || getBrowser() === "iOS") {
-        var elements = document.querySelectorAll(elementId);
-        elements.forEach((el) => {
-          el.style.display = show ? "block" : "none";
-        });
-      }
-    }
-    try {
-      showHideElementForBrowser(".desktop-device", false);
-      showHideElementForBrowser(".mobile-device", true);
-    } catch (e) {
-      console.log("Error adding event listener to header branding", e);
-    }
-
-    /**
-     * Initialize the Websocket
-     */
-    console.log("init");
-    const socket = io(location.host, {
-      path: "/ws/socket.io",
-      autoConnect: false,
-    });
+    </div>
 
-    socket.on("connect", () => {
-      socket.emit("initialize", { pid: "{{pid}}" });
-    });
+    <script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
+  </body>
+  <!-- Components -->
+  <script type="text/x-template" id="status-description">
```

Comment:
Can these not be imported from separate files? Just taking a brief look at: https://vuejs.org/guide/extras/web-components.html#sfc-as-custom-element if there's any relevance here.

Reply:
This was more a requirement we went with to simplify the situation when it came to how the template is used by the python code. We definitely could split it off but would need to handle the template expansion aka all the `{{add_asset("hourglass.svg")}}` which are being expanded by https://github.com/bcgov/vc-authn-oidc/blob/embedded-vue/oidc-controller/api/templates/helpers.py#L2

Reply:
Correct me if I am wrong @loneil @esune . I easily could have misunderstood the plans for this.

Reply:
I could try and change [get_authorize](https://github.com/bcgov/vc-authn-oidc/blob/embedded-vue/oidc-controller/api/routers/oidc.py#L86-L212) to support multiple files.

Reply:
The goal is to have this be contained and served in the html file so that shouldn't entail having to have anything built with Vite/etc or using `.vue` SFCs that need building.

Reply:
Ok In that case this seems like a good approach.